static/js/dashboard_routes.js

```diff
@@ -39,6 +41,10 @@ export function makeDashboardRoutes(browserHistory, store, onRouteUpdate) {
               <Route path="privacy" component={PrivacyTab} />
             </Route>
             <Route path="/terms_of_service" component={TermsOfServicePage} />
+            <Route path="/settings" component={SettingsPage} >
```

Comment:
`/settings/<username>` doesn't make sense here since we only want to view the settings for the current user. Why not just `/settings`?